storage/paths/local.go

```diff
@@ -462,7 +462,7 @@ func (st *Local) reportStorage(ctx context.Context) {
 	}
 }
 
-func (st *Local) Reserve(ctx context.Context, sid storiface.SectorRef, ft storiface.SectorFileType, storageIDs storiface.SectorPaths, overheadTab map[storiface.SectorFileType]int, minFreePercentage float64) (userRelease func(), err error) {
+func (st *Local) Reserve(ctx context.Context, sid storiface.SectorRef, ft storiface.SectorFileType, storageIDs storiface.SectorPaths, overheadTab map[storiface.SectorFileType]int, minFreePercentage float64) (release func(), err error) {
```

Comment:
This is very wrong, I'd either not name those returns or would keep the userRelease name.

If we name this `return` and change the `release := func() {}` line below to `=` you'll get panics on error - because the func will return `nil, err..`, then the defer in L478 will run a nil `release` (which was just set to nil in the return..)